tensorrt_llm/_torch/auto_deploy/utils/node_utils.py

```
@@ -442,3 +442,51 @@ def _get(name):
+            if exclude and exclude(arg):
+                continue
+
+            if depth > 1:
+                preds.extend(predecessors(arg, depth - 1, include, exclude))
+    return preds
+
+
+def successors(
+    node: Node,
+    depth: int = 1,
+    include: Callable[[Node], bool] = None,
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)